js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetChecks.tsx

```diff
@@ -38,93 +63,401 @@ export const AssetChecks = ({
+                          </Box>
+                          <Box padding={{horizontal: 24}}>
+                            <Caption
+                              color={Colors.textLight()}
+                              style={{textTransform: 'capitalize'}}
+                            >
+                              {assetCheckStatusDescription(check)}
+                            </Caption>
+                          </Box>
+                        </Box>
+                      </CheckRow>
+                    );
+                  })}
+                </Inner>
+              </Container>
+            </FixedScrollContainer>
+          </Box>
+        </Box>
+        <Box flex={{direction: 'column'}} style={{flex: 1}}>
+          <Box
+            style={{height: 56}}
+            border="bottom"
+            flex={{direction: 'row', alignItems: 'center', justifyContent: 'space-between'}}
+            padding={{vertical: 12, horizontal: 24}}
+          >
+            <Box flex={{direction: 'row', gap: 6, alignItems: 'center'}}>
+              <Icon name="asset_check" />
+              <Subtitle1>{selectedCheck.name}</Subtitle1>
+            </Box>
+            <ExecuteChecksButton assetNode={assetNode} checks={[selectedCheck]} label="Execute" />
+          </Box>
+          <Box
+            flex={{grow: 1, direction: 'column', gap: 12}}
+            padding={{horizontal: 24, vertical: 12}}
+          >
+            <CollapsibleSection
+              header={<Subtitle2>About</Subtitle2>}
+              headerWrapperProps={headerWrapperProps}
+            >
+              <Box padding={{top: 12}} flex={{gap: 12, direction: 'column'}}>
+                <Body2>
+                  {selectedCheck.description ?? (
+                    <Caption color={Colors.textLight()}>No description provided</Caption>
+                  )}
+                </Body2>
+                {/* {selectedCheck.dependencies?.length ? (
+                  <Box flex={{direction: 'row', gap: 6}}>
+                    {assetNode.dependencies.map((dep) => {
+                      const key = dep.asset.assetKey;
+                      return (
+                        <Link to={assetDetailsPathForKey(key)} key={tokenForAssetKey(key)}>
+                          <Tag icon="asset">{displayNameForAssetKey(key)}</Tag>
+                        </Link>
+                      );
+                    })}
+                  </Box>
+                ) : (
+                  <Caption color={Colors.textLight()}>No dependencies</Caption>
+                )} */}
+              </Box>
+            </CollapsibleSection>
+            <CollapsibleSection
+              header={<Subtitle2>Latest execution</Subtitle2>}
+              headerWrapperProps={headerWrapperProps}
+            >
+              <Box padding={{top: 12}}>
+                <div style={{display: 'grid', gridTemplateColumns: '1fr 1fr 1fr', gap: 24}}>
+                  <Box flex={{direction: 'column', gap: 5}}>
```

Comment:
More of a Josh thing (and def not worth changing) but I think in the asset overview page we have a similar-ish UI, but the gap between the items is 6  😂